docs/docs/Developing_with_JAC/Overview.md

```diff
@@ -9,26 +9,26 @@ Jaseci is a computational Model and language in one.Jaseci provides the ability
 
 ## Feature of Jaseci
 
-1. <strong>Easy to learn</strong> :
+1.<strong>Easy to learn: </strong> :
 Jaseci is a high level programming language. It is very easy to learn. The paradigm for Jaseci is based upon graphs.Once you underastand this new paradigm the language is simple.
 
-2. <strong>Free and Open Source </strong>
+2.<strong>Free and Open Source: </strong>
 Jaseci is completely free. Jaseci can be installled by runnning a simple pip command in your terminal.The Source code is avaliable here.
```

Comment:
Jaseci is a completely free and open source ecosystem that you can build on to include additional specialized libraries and engines to suit your needs for personal use or that of your clients.